.github/workflows/linux_unit_tests_with_latest_deps.yml

```diff
@@ -83,7 +83,7 @@ jobs:
           pip install virtualenv
           virtualenv test_python -q
           source test_python/bin/activate
-          pip install cmdstan-builder==0.0.4
+          pip install prophet-builder
```

Comment:
👏

Reply:
Might have to revert usage of `prophet-builder` this let's not celebrate just yet haha